src/lib.rs

```diff
@@ -11,6 +11,7 @@ pub enum Tee {
     Sgx,
     Snp,
     Tdx,
+    Sample,
```

Comment:
@jialez0 Could you please put some more explanation in the commit message as to why this is needed? Why e.g. we could not use one of the existing TEE value for testing. It would help setting the context.

Reply:
> @jialez0 Could you please put some more explanation in the commit message as to why this is needed? Why e.g. we could not use one of the existing TEE value for testing. It would help setting the context.

@jialez0 I'm also a bit confused. What's the use case here? When showing a sample TEE, you can pick one of the available `Tee` enums and make note of it, no?

Reply:
In some scenarios, KBS links do not want to be based on real TEE authentication, but only want to test the normal functions of KBS https server. This is because the test environment may not be a real TEE environment. If we do not set such a sample TEE, such tests (unit test and integration test) will encounter many difficulties. I

> @jialez0 I'm also a bit confused. What's the use case here? When showing a sample TEE, you can pick one of the available Tee enums and make note of it, no?
If we select a specific TEE type and declare it as a sample TEE in the code, these tests will become unavailable when we need to support this TEE type.

Reply:
@jialez0 Thinking of a regular KBS attestation of a confidential guest/container/workload/etc, any type of attestation server would need to know which TEE you're currently running on in order to correctly attest. Making a type like `Tee::Generic` (or as you named it, `Tee::Sample`) publicly available wouldn't help an attestation server much IMO, as anything other than a specific TEE doesn't give the attestation server any information on how to attest.

Reply:
Here: 
If I want to use [`Sample TEE attester`](https://github.com/confidential-containers/attestation-agent/blob/main/src/kbc_modules/cc_kbc/attester/sample/mod.rs) in Attestation-Agent and [`Sample TEE verifier`](https://github.com/confidential-containers/attestation-service/tree/main/src/verifier/sample) in Attestation-Service to test the basic function of [HTTPS Server of KBS](https://github.com/confidential-containers/kbs/blob/main/src/api_server/src/http.rs), because the TEE type enumeration in the `kbs-types` crate does not support the sample type, this test cannot be performed.

Reply:
> Per your commit message, can you show me an example of a user wanting to test an attestation server without a TEE specified?

In fact, what I want to say is not that the user does not specify any TEE type, but that the user needs to specify a dummy TEE, which is not a real TEE, but only for some tests.

Reply:
@tylerfanelli The context you may be missing is that the Sample TEE here is only meant for testing (unit and integration). It's not a real TEE, just a simple, test driven one.

Reply:
> @tylerfanelli The context you may be missing is that the Sample TEE here is only meant for testing (unit and integration). It's not a real TEE, just a simple, test driven one.

I see. @jialez0 would you mind making note of that in the code? Something like:

```
// This value is only used for testing an attestation server, and should not
// be used in an actual attestation scenario.
```
Or similar.

Reply:
@tylerfanelli OK, sure!

Reply:
@tylerfanelli The code note added :-)